tests/pytests/unit/modules/test_tls.py

```diff
@@ -0,0 +1,99 @@
+import os
+
+import pytest
+import salt.modules.tls as tls
+from tests.support.mock import MagicMock, patch
+
+_TLS_TEST_DATA = {
```

Comment:
Why do you need this dictionary to be a global?
If you use it multiple times, make it a fixture.
If you only need to create it once, make the fixture module scoped.

Reply:
Good point, I've tweaked the tests to use a fixture now.